src/support.js

```diff
@@ -0,0 +1,217 @@
+    parsers: ["babylon", "flow"],
+    group: "JavaScript",
+    extensions: [".jsx"],
+    tmScope: "source.js.jsx",
+    aceMode: "javascript",
+    codemirrorMode: "jsx",
+    codemirrorMimeType: "text/jsx",
+    liguistLanguageId: 178,
+    vscodeLanguageIds: ["javascriptreact"]
+  },
+  {
+    name: "TypeScript",
+    since: "1.4.0",
+    parsers: ["typescript"],
+    group: "JavaScript",
+    aliases: ["ts"],
+    extensions: [".ts", ".tsx"],
+    tmScope: "source.ts",
+    aceMode: "typescript",
+    codemirrorMode: "javascript",
+    codemirrorMimeType: "application/typescript",
+    liguistLanguageId: 378,
+    vscodeLanguageIds: ["typescript", "typescriptreact"]
+  },
+  {
+    name: "CSS",
+    since: "1.4.0",
+    parsers: ["css"],
+    group: "CSS",
+    tmScope: "source.css",
+    aceMode: "css",
+    codemirrorMode: "css",
+    codemirrorMimeType: "text/css",
+    extensions: [".css"],
+    liguistLanguageId: 50,
+    vscodeLanguageIds: ["css"]
+  },
+  {
+    name: "Less",
+    since: "1.4.0",
+    parsers: ["less"],
+    group: "CSS",
+    extensions: [".less"],
+    tmScope: "source.css.less",
+    aceMode: "less",
+    codemirrorMode: "css",
+    codemirrorMimeType: "text/css",
+    liguistLanguageId: 198,
+    vscodeLanguageIds: ["less"]
+  },
+  {
+    name: "SCSS",
+    since: "1.4.0",
+    parsers: ["scss"],
+    group: "CSS",
+    tmScope: "source.scss",
+    aceMode: "scss",
+    codemirrorMode: "css",
+    codemirrorMimeType: "text/x-scss",
+    extensions: [".scss"],
+    liguistLanguageId: 329,
+    vscodeLanguageIds: ["scss"]
+  },
+  {
+    name: "GraphQL",
+    since: "1.5.0",
+    parsers: ["graphql"],
+    extensions: [".graphql", ".gql"],
+    tmScope: "source.graphql",
+    aceMode: "text",
+    liguistLanguageId: 139,
+    vscodeLanguageIds: ["graphql"]
+  },
+  {
+    name: "JSON",
+    since: "1.5.0",
+    parsers: ["json"],
+    group: "JavaScript",
+    tmScope: "source.json",
+    aceMode: "json",
+    codemirrorMode: "javascript",
+    codemirrorMimeType: "application/json",
+    extensions: [
+      ".json",
+      ".json5",
+      ".geojson",
+      ".JSON-tmLanguage",
+      ".topojson"
+    ],
+    filenames: [
+      ".arcconfig",
+      ".jshintrc",
+      ".babelrc",
+      ".eslintrc",
+      ".prettierrc",
+      "composer.lock",
+      "mcmod.info"
+    ],
+    linguistLanguageId: 174,
+    vscodeLanguageIds: ["json"]
+  },
+
+  {
+    name: "Markdown",
+    since: "1.8.0",
+    parsers: ["markdown"],
+    aliases: ["pandoc"],
+    aceMode: "markdown",
+    codemirrorMode: "gfm",
+    codemirrorMimeType: "text/x-gfm",
+    wrap: true,
+    extensions: [
+      ".md",
+      ".markdown",
+      ".mdown",
+      ".mdwn",
+      ".mkd",
+      ".mkdn",
+      ".mkdown",
+      ".ron",
+      ".workbook"
+    ],
+    filenames: ["README"],
+    tmScope: "source.gfm",
+    linguistLanguageId: 222,
+    vscodeLanguageIds: ["markdown"]
+  },
+  {
+    name: "HTML",
+    since: undefined, // unreleased
```

Comment:
No plans to release this yet (in eg 1.8.0), right?

Reply:
Probably not. Depends on #2976